kong/plugins/jwt/schema.lua

```diff
@@ -8,6 +9,13 @@ local function check_user(anonymous)
   return false, "the anonymous user must be empty or a valid uuid"
 end
 
+local function check_maximum_expiration_positive(v)
+  if v < -1 or v == 0 then
+    return false, "maximum_expiration should be -1 or greater than 0"
```

Comment:
No need to repeat the name of the property in this error message, it will already be contained in the key for this error. We are also lacking a test for this validation.